docs/tutorials/typescript/digital-ocean-deployment/index.mdx

```diff
@@ -0,0 +1,283 @@
+---
+id: index
+title: Deploy a TypeScript application to production on a Digital Ocean Droplet
+description: Learn how to deploy your TypeScript-based Temporal Application to a DigitalOcean Droplet using Ubuntu 20.04 or 22.04
+sidebar_label: Deploy a TypeScript application to production on a Digital Ocean Droplet
+tags:
+  - typescript
+  - deployment
+  - digital-ocean
+  - temporal-cloud
+keywords:
+  - TypeScript Temporal Application deployment
+  - DigitalOcean Droplet setup
+  - Ubuntu 20.04 server setup
+  - Ubuntu 22.04 server setup
+  - Install Node.js on Ubuntu
+  - Install TypeScript on Ubuntu
+  - SSH setup DigitalOcean
+  - Custom domain configuration
+  - Nginx reverse proxy setup
+  - Let's Encrypt SSL certificate
+  - PM2 process manager
+  - Temporal Cloud integration
+  - Deploy TypeScript application DigitalOcean
+  - Temporal Cloud certificates
+---
+
+When you are ready to deploy your TypeScript based Temporal Application to production, you need a server to host it.
+Digital Ocean is provides flexible Cloud servers called Droplets that can be used to host your application.
+
+In this tutorial we will walk through the steps to deploy your Temporal Application to a Digital Ocean Droplet while using Temporal Cloud as your Temporal Application's supervisor.
+
+## Create a Droplet {#create-droplet}
+
+Assuming you already have a Digital Ocean Account, create a new Droplet and choose the Ubuntu 20.04 image or Ubuntu 22.04 image.
+
+**Set up SSH**
+
+Next, set up SSH so you can set up your server from your local terminal.
+You can do that by following the [Add SSH Key to Droplet tutorial](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/).
+
+If you already have a private key on your local machine and you need to force SSH to use the new key, you can force it with the `-i` flag, for example:
+
+```
+ssh -i ~/.ssh/id_rsa_digitalocean
+```
+
+**Install Node.js**
+
+Once you have SSH'd into your Droplet, you can install Node.js.
+
+Follow these tutorials, for Ubuntu 20.04 or 22.04 respectively:
+
+- [How To Install Node.js on Ubuntu 20.04](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04)
+- [How To Install Node.js on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-22-04)
+
+**Install TypeScript and TS Node**
+
+```
+sudo npm install -g typescript ts-node
+```
+
+## Use a custom domain {#custom-domain}
+
+You need a domain, because you need to have a valid SSL certificate for the Temporal Application to communicate with Temporal Cloud.
+
+Register your domain and ensure the domain's A record points to the droplet's IP address.
+
+## Set up Nginx as a reverse proxy {#set-up-nginx}
```

Comment:
```suggestion
## Step 2 -- Install and Configure Nginx as a Reverse Proxy {#set-up-nginx}
```

After looking at how our h2s are displayed in Docusaurus, I think we absolutely need some kind of numbering -- otherwise it makes it much too hard (imo) to read these pages linearly, they wind up looking like reference documentation. I think this would technically be Step 2, if Step 1 is installing Nodejs and everything else is prerequisite.